Mathlib/RingTheory/DedekindDomain/AdicValuation.lean

```diff
@@ -472,4 +507,32 @@ instance AdicCompletion.instIsScalarTower' :
 
 end AlgebraInstances
 
+open nonZeroDivisors algebraMap in
+variable {R K} in
+lemma clear_local_denominator (v : HeightOneSpectrum R)
```

Comment:
Can you find a more descriptive name?

Reply:
I've gone for `adicCompletion.mul_nonZeroDivisor_mem_adicCompletionIntegers`. What do you think?